docstore/mongodocstore/mongo_test.go

```diff
@@ -182,12 +193,12 @@ func newTestClient(t *testing.T) *mongo.Client {
 func BenchmarkConformance(b *testing.B) {
 	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
 	defer cancel()
-	client, err := Dial(ctx, serverURI)
+	client, err := Dial(ctx, serverURIV4)
```

Comment:
Why only v4?

Reply:
This is a benchmark so there isn't compatibility concern.

Reply:
Add a comment?

Reply:
Done.